commands/device/uninstall-application.ts

```diff
@@ -10,12 +10,10 @@ export class UninstallApplicationCommand implements ICommand {
 
 	public execute(args: string[]): IFuture<void> {
 		return (() => {
-			this.$devicesServices.initialize({ deviceId: this.$options.device, skipInferPlatform: true }).wait();
+			this.$devicesService.initialize({ deviceId: this.$options.device, skipInferPlatform: true }).wait();
 
-			let action = (device: Mobile.IDevice) =>  {
-				return (() => device.applicationManager.uninstallApplication(args[0]).wait()).future<void>()();
-			};
-			this.$devicesServices.execute(action).wait();
+			let action = (device: Mobile.IDevice) =>  { return (() => device.applicationManager.uninstallApplication(args[0]).wait()).future<void>()(); };
```

Comment:
Use `let action = (device: Mobile.IDevice) => device.applicationManager.uninstallApplication(args[0]);` - No need to create an extra future.